packages/program-boilerplate/src/conversion.ts

```diff
@@ -0,0 +1,167 @@
+/**
+ * @module conversion
+ */
+
+/**
+ * A custom field based conversion rule
+ */
+type CustomFieldConversionRule = {
+  operator: number | string;
+  fieldName: string;
+  value: RuleValue;
+};
+
+/**
+ * An event-based conversion rule
+ */
+type EventTriggerRule = {
+  operator: string;
+  eventKey: string;
+  value: string;
+};
+
+/**
+ * A key-value pair
+ */
+type Event = {
+  key: string;
+  fields: {
+    [key: string]: any;
+  };
+};
+
+/**
+ * A user object returned from a GraphQL query
+ */
+type UserQueryResult = {
+  customFields: {
+    [key: string]: any;
+  };
+};
+
+/**
+ * The value of a rule
+ */
+type RuleValue = string | boolean | number;
+
+//functions
+/**
+ * Turns a string scalar into a Number, Boolean or String
+ *
+ * @param {string} value the value of a rule passed in as a string
+ * @returns {RuleValue} The parsed value
+ */
+function parseValue(value: string): RuleValue {
+  if (/^(-|\+)?([0-9]+(\.[0-9]+)?)$/.test(value)) {
+    return Number(value);
+  }
+
+  if (value.toLowerCase() === 'true' || value.toLowerCase() === 'yes') {
+    return Boolean(true);
```

Comment:
Why not just `true`?

Reply:
Again, I'm not sure as I didn't write that code